src/bin/ledger-tool.rs

```diff
@@ -136,6 +155,9 @@ fn main() {
                 if i >= head {
                     break;
                 }
+                if entry.num_hashes < min_hashes {
```

Comment:
if you continue from here, verification will fail on the next entry  "verify" can't skip entries

Reply:
That did occur to me, but I tested it with the ledgers generated by the unit tests, and it didn't report any failures.  So, not sure what's up with that, but I'll fix this and send a new PR.

Reply:
likely the unit tests you sampled aren't generating ledgers with ticks...

can you add a mod test {} to this file to get at all these cases?

Reply:
WWAkRaniD?

Reply:
Ideally both, but it will take a little while, since I'm somewhat new at Rust and very much still learning this code.